packages/amplify_core/lib/src/types/notifications/push/push_notification_message.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_core/amplify_core.dart';
+
+class PushNotificationMessage with AWSDebuggable {
```

Comment:
If you'd like to make the types "printable", lets use `AWSSerializable` mixin to and implement `toJson` instead of `toString`. (same suggestion to other types which have overridden `toString`)

```suggestion
class PushNotificationMessage with AWSDebuggable, AWSSerializable<Map<String, Object?>> {
```